src/UI/citizens/components/header/CitizensHeaderStyle.js

```diff
@@ -1,12 +1,353 @@
 import styled from 'styled-components';
 
 const HeaderContainer = styled.header`
-  width: 100%;
+  width: 1440px;
   padding: 20px;
 `;
 
 const HeaderTitle = styled.h1`
-  color: red;
+  width: 576px;
+  height: 40px;
+  font-family: Lora;
+  font-size: 32px;
+  font-weight: bold;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #000000;
+`;
+
+const HeaderButton = styled.div`
+  width: 108px;
+  height: 24px;
+  font-family: Poppins;
+  font-size: 14px;
+  font-weight: 500;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0f4850;
+`;
+
+const HeaderImage = styled.img`
+  width: 100%;
+  height: 336px;
+  object-fit: cover;
+  top center / 200px 200px;
+  `;
+
+const RezervacijaDvorana = styled.div`
```

Comment:
Neću sada pisati detaljne komentare za layout jer smo to prokomentirali, ali probaj izbjegavati upotrebu position: absolute-a i pozicioniraj elemente sa hijerarhijama div-ova, flex-boxom, marginama i paddingom